test/unit/bootloader/config/base_test.py

```diff
@@ -10,6 +13,10 @@
 
 
 class TestBootLoaderConfigBase:
+    @fixture(autouse=True)
```

Comment:
Nice! Learned something new today :)

Reply:
I also learned a lot while doing this :-)